### DISCUSSION TITLE
Headphone Jack board

### DISCUSSION THREAD

**Discussion by Schnappis** [PE5PVB/SI4684-DAB-Receiver#13] [Q&A]
Hello Sjef
What board is used for the Headphone Jack?
Do you have a Link where it can be bought?
I could not find it in the BOM.
Thanks!

**Comment by PE5PVB** [comment]
https://www.tinytronics.nl/nl/kabels-en-connectoren/kabels-en-adapters/audio/audiokabels/3.5mm-female-jack-board-met-kabel

**Reply by Schnappis** [reply]
Thank you!
unfourtunately, this shop does not ship to my Country.
Do you know of any alternatives?

**Comment by PE5PVB** [comment]
No. I can't help you with that.
